### ISSUE TITLE
Discrepancy in "noeject" presence between Isolinux and Grub2 menus

### ISSUE THREAD

**Issue by Joey3000** [stevenshiau/clonezilla#20]
Currently, the Linux kernel parameters in `/syslinux/isolinux.cfg` (for Isolinux) and `/EFI/boot/grub.cfg` (for Grub2) of a release are equal, except for the presence of the `noeject` parameter. It is absent in the former and present in the latter. Is there a reason for this discrepancy?

clonezilla-live-2.4.5-7-amd64

**Comment by stevenshiau** [comment]
That's true. They should be consistent. Did you try to put "noeject" in your isolinux.cfg and test if it has any effect when rebooting or halting? IIRC, it seems this parameter has lost its function in the live-boot used in Clonezilla live.
